src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -28,6 +28,8 @@ def initialize()
         config_param :conn_retry_timeout_ms, :integer, :default => 60000
         desc 'the field name for the event emit time stamp'
         config_param :emit_timestamp_name, :string, :default => "FluentdIngestTimestamp"
+        desc 'the maximum record size to emit'
```

Comment:
>  desc 'the maximum record size to emit' [](start = 6, length = 40)

the maximum record size supported by mdsd agent is 128*1024-1. by making 'max_record_size' as a parameter, this gives the user an impression that it can support record size bigger than this. I recommend the following solution
1) define a global variable for absolute record size in "initialize()". assuming it is called 'mdsd_max_record_size'.
2) make it clear in 'max_record_size' parameter's description that it can never exceed the number in #1.
3) in function record_too_large, use min(mdsd_max_record_size, max_record_size) when checking whether the msg needs to be dropped